tests/src/performance/Scenario/JitBench/JitBenchHarness.cs

```diff
@@ -0,0 +1,487 @@
+
+            ++s_iteration;
+        }
+
+        private static ScenarioBenchmark PostProcessing()
+        {
+            PrintHeader("Starting POST");
+
+            var scenarioBenchmark = new ScenarioBenchmark("MusicStore") {
+                Namespace = "JitBench"
+            };
+
+            // Create (measured) test entries for this scenario.
+            var startup = new ScenarioTestModel("Startup time");
+            scenarioBenchmark.Tests.Add(startup);
+
+            var request = new ScenarioTestModel("Request time");
+            scenarioBenchmark.Tests.Add(request);
+
+            // Add measured metrics to each test.
+            startup.Performance.Metrics.Add(new MetricModel {
+                Name = "Duration",
+                DisplayName = "Duration",
+                Unit = "ms"
+            });
+            request.Performance.Metrics.Add(new MetricModel {
+                Name = "Duration",
+                DisplayName = "Duration",
+                Unit = "ms"
+            });
+
+            for (int i = 0; i < s_ScenarioConfiguration.Iterations; ++i)
+            {
+                var startupIteration = new IterationModel { Iteration = new Dictionary<string, double>() };
+                startupIteration.Iteration.Add("Duration", s_startupTimes[i]);
+                startup.Performance.IterationModels.Add(startupIteration);
+
+                var requestIteration = new IterationModel { Iteration = new Dictionary<string, double>() };
+                requestIteration.Iteration.Add("Duration", s_requestTimes[i]);
+                request.Performance.IterationModels.Add(requestIteration);
+            }
+
+            return scenarioBenchmark;
+        }
+
+        private static void LaunchProcess(ProcessStartInfo processStartInfo, int timeoutMilliseconds, IDictionary<string, string> environment = null)
+        {
+            Console.WriteLine();
+            Console.WriteLine($"{System.Security.Principal.WindowsIdentity.GetCurrent().Name}@{Environment.MachineName} \"{processStartInfo.WorkingDirectory}\"");
+            Console.WriteLine($"[{DateTime.Now}] $ {processStartInfo.FileName} {processStartInfo.Arguments}");
+
+            if (environment != null)
+            {
+                foreach (KeyValuePair<string, string> pair in environment)
+                {
+                    if (!processStartInfo.Environment.ContainsKey(pair.Key))
+                        processStartInfo.Environment.Add(pair.Key, pair.Value);
+                    else
+                        processStartInfo.Environment[pair.Key] = pair.Value;
+                }
+            }
+
+            using (var p = new Process() { StartInfo = processStartInfo })
+            {
+                p.Start();
+                if (p.WaitForExit(timeoutMilliseconds) == false)
+                {
+                    // FIXME: What about clean/kill child processes?
+                    p.Kill();
```

Comment:
Good call. Given that this is likely something we need across many scenario benchmarks in the future, though, would it make sense to add it as publicly-available utility functionality in XUnit-Performance itself? (We can deprecate it in the future if a future version of the framework picks up support for doing a clean `KillAll()`).